Update documentation for automatic model undeployment (#7083)
opensearch-project/documentation-website: _ml-commons-plugin/remote-models/index.md

To learn how to use the model for vector search, see [Using an ML model for neural search]({{site.url}}{{site.baseurl}}/search-plugins/neural-search/#using-an-ml-model-for-neural-search).

## Step 7 (Optional): Undeploy the model 

> Undeploy the model automatically by defining a TTL in the Model or use the Undeploy API to undeploy the model manually. To learn more about Model undeploy, see the [Undeploy API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/undeploy-model/).

Reviewer: ```suggestion
Undeploy the model automatically by defining a TTL in the model settings or use the Undeploy API to undeploy the model manually. To learn more about model undeploy, see [Undeploy API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/undeploy-model/).
```

> Undeploy the model automatically by defining a TTL in the model settings or use the Undeploy API to undeploy the model manually. To learn more about model undeploy, see [Undeploy API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/undeploy-model/).

Reviewer 2: ```suggestion
You can undeploy the model automatically by defining a TTL in the model settings or by using the Undeploy API to undeploy the model manually. For more information, see [Undeploy API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/undeploy-model/).
```
